orte/mca/ess/base/ess_base_std_tool.c

```diff
@@ -243,7 +231,7 @@ int orte_ess_base_tool_finalize(void)
 
     /* release the event base */
     if (progress_thread_running) {
-        opal_stop_progress_thread("orte", true);
+        opal_progress_thread_finalize("orte");
```

Comment:
I think this line should be removed - we removed the progress_thread create up above, so this "orte" thread will never be started in the tool.